reminders.txt

```diff
@@ -26,3 +26,4 @@ Hey human, pursue for excellence and success wil chase you ✨.
 Hey human, remember to eat some fruits and stay healthy ok
 Hey human, if you feel stressed, it's ok to reach out to your loved ones :)
 Hey human, remember to be patient, good things take time.
+Hey human, don't learn alone. Learn in public and share the things you learn and this will keep you motivated 
```

Comment:
Let's phrase it slightly differently to something like:

Hey human, if you're learning something, you might want to share what you learn online! That way, it'll keep you motivated and other people can learn from you

Reply:
> Let's phrase it slightly differently to something like:
> 
> Hey human, if you're learning something, you might want to share what you learn online! That way, it'll keep you motivated and other people can learn from you

Yes! I was just going to comment something like this.

Reply:
That's nice